server/src/main/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthLogoutHandler.java

```diff
@@ -123,4 +123,11 @@ private String getZoneDefaultUrl() {
     }
     return config.getLinks().getLogout().getRedirectUrl();
   }
+
+  public Boolean getPerformRpInitiatedLogout(AbstractExternalOAuthIdentityProviderDefinition oauthConfig) {
```

Comment:
It looks like this `Boolean` could actually be a `boolean`. That would make sure that it's never `null`.

Reply:
You are right! commit pushed.